private/protocol/rest/v2/utils_test.go

```diff
@@ -0,0 +1,36 @@
+package rest
```

Comment:
What do you think about naming this `shared_test.go`. would follow similar testing behavior/logic that is shared between multiple test files.